src/options/SMTConfig.h

```diff
@@ -158,22 +225,14 @@ typedef struct SpPref    { int t; } SpPref;
 
 typedef struct SpFormat  { int t; } SpFormat;
 
-struct ItpAlgorithm { int x; bool operator==(const ItpAlgorithm& o) const { return x == o.x; }};
-static const struct ItpAlgorithm itp_alg_mcmillan  = { 0 };
-static const struct ItpAlgorithm itp_alg_pudlak    = { 1 };
-static const struct ItpAlgorithm itp_alg_mcmillanp = { 2 };
-static const struct ItpAlgorithm itp_alg_ps        = { 3 };
-static const struct ItpAlgorithm itp_alg_psw       = { 4 };
-static const struct ItpAlgorithm itp_alg_pss       = { 5 };
-static const struct ItpAlgorithm itp_euf_alg_strong  = { 0 };
-static const struct ItpAlgorithm itp_euf_alg_weak  = { 2 };
-static const struct ItpAlgorithm itp_euf_alg_random  = { 3 };
-static const struct ItpAlgorithm itp_lra_alg_strong  = { 0 };
-static const struct ItpAlgorithm itp_lra_alg_weak  = { 2 };
-static const struct ItpAlgorithm itp_lra_alg_factor  = { 3 };
-static const struct ItpAlgorithm itp_lra_alg_decomposing_strong  = {4 };
-static const struct ItpAlgorithm itp_lra_alg_decomposing_weak  = {5 };
-static const char *itp_lra_factor_0 = "1/2";
+enum class ItpAlgorithm {
+    itp_alg_mcmillan, itp_alg_pudlak, itp_alg_mcmillanp,
+    itp_alg_ps, itp_alg_psw, itp_alg_pss, itp_euf_alg_strong,
+    itp_euf_alg_weak, itp_euf_alg_random, itp_lra_alg_strong,
+    itp_lra_alg_weak, itp_lra_alg_factor, itp_lra_alg_decomposing_strong,
+    itp_lra_alg_decomposing_weak};
```

Comment:
Maybe this `enum` should be split into separate `enums` for propositional, euf and lra interpolation algorithms?